API-reference.md

```diff
@@ -1,7 +1,7 @@
 ### trackPageView
 
 ```ts
-applicationInsights.trackPageView(pageView: IPageViewTelemetry, customProperties?: { [key: string]: any })
```

Comment:
just curious, why are we removing `customProperties?` from readme? Isn't it still a param in code? 🤔

Reply:
It is only used for 1P SDK and not meant to be exposed in this SDK's APIs. Although we still need to handle it for internal users, I think its best we just pick one way to do it instead of creating ambiguity and confusion with 2 (e.g. is there a difference between passing in one way or the other?) [Node.js SDK API](https://github.com/microsoft/ApplicationInsights-node.js/blob/be0f918c77aa7f0895f2784fa8c74ebc29131472/Declarations/Contracts/TelemetryTypes/Telemetry.ts#L12) uses `properties` named argument already, so I'm leaning that way. WDYT?

Reply:
Ok, makes sense. 👍